src/libs/ajax.js

```diff
@@ -1122,7 +1122,8 @@ const Dockstore = signal => ({
 
 const Martha = signal => ({
   getDataObjectMetadata: async url => {
-    const res = await fetchMartha('martha_v2', _.mergeAll([jsonBody({ url }), appIdentifier, { signal, method: 'POST' }]))
+    // TODO: switch to 'martha_v3' to point to updated endpoint
```

Comment:
Is there a reason not to check in `martha_v3` today? It is in production in all environments.

Reply:
doing this to follow the model we've used for other codebases that are switching from `martha_v2` to `martha_v3': first make sure code supporting both works, then do the switch-over